posts/app.py

```diff
@@ -81,8 +98,10 @@ def upload_new_post_to_db(post, collection):
                              "attributes {required_attributes}")
     if not post["text"] and not post["files"]:
         raise ValueError("One of text or files must not be empty.")
-    # post is valid, add on timestamp and insert into db
+    # post is valid, add on timestamp, upload files, insert into db
     post["created_at"] = generate_timestamp()
+    post["files"] = [
```

Comment:
Update comment on Line 84 to reflect list of urls

Reply:
Done https://github.com/knative-portability/large-events/pull/20/commits/e0c5ac22af03a910c3995786f917a0ea206258db